### ISSUE TITLE
overfitting

### ISSUE THREAD

**Issue by zhongqiang92** [clovaai/deep-text-recognition-benchmark#154]
hi,can you tell me how to solve overfitting

**Comment by YacobBY** [comment]
The first option would be to adjust the learning rate of the algorithm but assuming you're talking about training on your own dataset I think some practical advice is more useful. 

You need to have a big enough dataset so that it doesn't try to optimize for the same images. If you don't have a massive dataset of text of your own, I would recommend using the existing datasets and adding your own dataset to it on the side. Then train it with a percentage of maybe ~4% for your own dataset using the --batch_ratio option.

**Comment by keepdevelopment** [comment]
> The first option would be to adjust the learning rate of the algorithm but assuming you're talking about training on your own dataset I think some practical advice is more useful.
> 
> You need to have a big enough dataset so that it doesn't try to optimize for the same images. If you don't have a massive dataset of text of your own, I would recommend using the existing datasets and adding your own dataset to it on the side.

thank you  very much,  I've tried using weight attenuation, and I' ve found that it's good for digital recognition

**Comment by Amithvassit** [comment]
Can you please how did you overcome overfitting

**Comment by macqueen09** [comment]
I hace almost 25W images to digital recognition by hand , 5W are real and others are Simulation。 It overfitting very fast.
both adam(0.001) and Adadelta(1) are overfitting very fast. 
could you have some Training strategy？

another question ， I couldn't find the code for data enhancement. Should I add them myself?

**Comment by zhongqiang92** [comment]
> I hace almost 25W images to digital recognition by hand , 5W are real and others are Simulation。 It overfitting very fast.
> both adam(0.001) and Adadelta(1) are overfitting very fast.
> could you have some Training strategy？
> 
> another question ， I couldn't find the code for data enhancement. Should I add them myself?
> Thanks very much

不好意思，是我表述有问题，我只是尝试使用更多样式的数据，以及尝试使用weightdecay,dropblock ，而且在少量的数据下测试，目测有用，并没有具体的统计数据
